### PR TITLE
[W4.11e][W13-4] Andy Chan Wai Yee

### PR DESCRIPTION
**Pull request by andlimey**

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -43,4 +42,31 @@ private void assertAreUnique(Object... objects) {
...
     }
+
+    private void assertIsNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
+    }
+
+    private void assertIsNotNull(Object... objects) {
+        assertFalse(Utils.isAnyNull(objects));
+    }
+
```
Comment (parvathy-sudhir-pillai) [inline test/java/seedu/addressbook/common/UtilsTest.java:53]: It's good to follow the style used by the code nearby 👍
Having said that, it is doubtful if extracting these two methods add any value.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -43,4 +42,31 @@ private void assertAreUnique(Object... objects) {
...
+        // Two Strings
+        assertIsNotNull("abc", "def");
+
+        // No string
+        assertIsNull((Object) null);
+
+        // One string
+        assertIsNull("abc", null);
+        assertIsNull(null, "abc");
```
Comment (parvathy-sudhir-pillai) [inline test/java/seedu/addressbook/common/UtilsTest.java:70]: May be add more tests for multiple null and non-null objects

**Review by parvathy-sudhir-pillai — COMMENTED** [review]
Good effort @andlimey. See my comments for suggestions. Please close this PR after reading comments.

**Review by parvathy-sudhir-pillai — APPROVED** [review]
